### PR TITLE
Note/git add u

### PR DESCRIPTION
**Pull request by KarthikSiddarth**
Added note on staging all tracked file at once without staging untracked file

### CODE REVIEW

**Comment by RichardLitt** [comment]
Might be worth explaining what happens if you use the `--update` flag with a pathspec, too.

**Comment by KarthikSiddarth** [comment]
add a note on staging with pathspec

File: README.md [hunk 1]
```diff
@@ -561,6 +564,24 @@ will do the rest of the work for you.
 
 ## Staging
 
+<a href="#i-want-to-stage-all-tracked-file-and-leave-untracked-files"></a>
+
+### I want to stage all tracked file and leave untracked files
```
Comment (RichardLitt) [inline README.md]: files

File: README.md [hunk 2]
```diff
@@ -561,6 +564,24 @@ will do the rest of the work for you.
...
+
+```sh
+(my-branch*)$ git add -u
+```
+
+#### To stage part of tracked files
+
+```sh
+to stage files with ext .txt
```
Comment (RichardLitt) [inline README.md]: Needs `#` for comments.

File: README.md [hunk 3]
```diff
@@ -561,6 +564,24 @@ will do the rest of the work for you.
...
+(my-branch*)$ git add -u *.txt
+
+to stage all files inside directory src
```
Comment (RichardLitt) [inline README.md]: Needs `#`

**Comment by KarthikSiddarth** [comment]
@RichardLitt @eamanu fixed all errors

File: README.md (hunk 3 above)
Comment (KarthikSiddarth) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (KarthikSiddarth) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (KarthikSiddarth) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -561,6 +564,24 @@ will do the rest of the work for you.
 
 ## Staging
 
+<a href="#i-want-to-stage-all-tracked-files-and-leave-untracked-files"></a>
+
+### I want to stage all tracked files and leave untracked files
+
+```sh
+(my-branch*)$ git add -u
```
Comment (RichardLitt) [inline README.md]: Remove `(my-branch*)` here and in the following two commands. It isn't necessary, and it doesn't follow the naming conventions for the rest of the document.

File: README.md [hunk 5]
```diff
@@ -1591,6 +1612,12 @@ $ rm -rf .git/modules/submodulename
 
 ## Miscellaneous Objects
 
+### Copy a folder or file from one branch to another
+
+```sh
+$ (my-branch)$ git checkout <branch-you-want-the-directory-from> -- <folder-name or file-name>
```
Comment (RichardLitt) [inline README.md]: Remove `$ (my-branch)`. It isn't necessary, and you've mistakenly added two dollar signs. Thanks.

Comment (KarthikSiddarth) [reply README.md]: sure

File: README.md (hunk 4 above)
Comment (KarthikSiddarth) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (KarthikSiddarth) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (KarthikSiddarth) [reply README.md]: done

**Comment by RichardLitt** [comment]
Thank you for being so patient! Merging.

**Pull request merged**